content/docs/getting_started/manual_demo.md

```diff
@@ -81,31 +81,32 @@ Read more on OSM's integrations with Prometheus, Grafana, and Jaeger in the [obs
+In this section we will deploy 5 different Pods, and we will apply policies to control the traffic between them.
 
-- `bookbuyer` is an HTTP client making requests to `bookstore`. This traffic is **permitted**.
+- `bookbuyer` is an HTTP client making requests to `bookstore-v1`. This traffic is **permitted**.
 - `bookthief` is an HTTP client and much like `bookbuyer` also makes HTTP requests to `bookstore`. This traffic should be **blocked**.
-- `bookstore` is a server, which responds to HTTP requests. It is also a client making requests to the `bookwarehouse` service.
-- `bookwarehouse` is a server and should respond only to `bookstore`. Both `bookbuyer` and `bookthief` should be blocked.
+- `bookstore-v1` is a server, which responds to HTTP requests. It is also a client making requests to the `bookwarehouse` service.
+- `bookwarehouse` is a server and should respond only to `bookstore-v1`. Both `bookbuyer` and `bookthief` should be blocked.
+- `mysql` is a MySQL database only reachable by `bookwarehouse`.
 
 
 We are going to craft SMI policies, which will bring us to this final desired
 state of allowed and blocked traffic between pods:
 
-| from  /   to: | bookbuyer | bookthief | bookstore | bookwarehouse |
-|---------------|-----------|-----------|-----------|---------------|
-| bookbuyer     |     \     |     ❌     |     ✔    |       ❌       |
-| bookthief     |     ❌     |     \     |     ❌     |       ❌       |
-| bookstore     |     ❌     |     ❌     |     \     |       ✔      |
-| bookwarehouse |     ❌     |     ❌     |     ❌     |      \        |
+| from  /   to: | bookbuyer | bookthief | bookstore-v1 | bookwarehouse | mysql |
+|---------------|-----------|-----------|--------------|---------------|-------|
+| bookbuyer     |     \     |     ❌    |     ✔        |      ❌       |  ❌   |
+| bookthief     |     ❌    |     \     |     ❌       |      ❌       |  ❌   |
+| bookstore-v1  |     ❌    |     ❌    |     \        |      ✔        |  ❌   |
+| bookwarehouse |     ❌    |     ❌    |     ❌       |      \        |  ✔    |
+| mysql         |     ❌    |     ❌    |     ❌       |      ❌       |  \    |
 
 
 To show SMI Traffic Split, we will deploy an additional application:
 
-- `bookstore-v2` - this is the same container as the first `bookstore` we deployed, but for this demo we will assume that it is a new version of the app we need to upgrade to.
+- `bookstore-v2` - this is the same container as the first `bookstore-v1` we deployed, but for this demo we will assume that it is a new version of the app we need to upgrade to.
 
-The `bookbuyer`, `bookthief`, `bookstore`, and `bookwarehouse` Pods will be in separate Kubernetes Namespaces with
-the same names. Each new Pod in the service mesh will be injected with an Envoy sidecar container.
+The `bookbuyer`, `bookthief`, `bookstore-v1`, and `bookwarehouse` Pods will be in separate Kubernetes Namespaces. Each new Pod in the service mesh will be injected with an Envoy sidecar container.
```

Comment:
We could note that the mysql pod will be deployed in the same namespace as bookwarehouse.